test/aspnet-core-demo/AbpAspNetCoreDemo/AbpAspNetCoreDemoModule.cs

```diff
@@ -41,6 +42,14 @@ public override void PreInitialize()
                 .CreateControllersForAppServices(
                     typeof(AbpAspNetCoreDemoCoreModule).GetAssembly()
                 );
+
+
+            Configuration.IocManager.Resolve<IAbpAspNetCoreConfiguration>().RouteConfiguration.Add(routes =>
```

Comment:
Use Configuration.Modules.AbpAspNetCore() instead of Configuration.IocManager.Resolve<IAbpAspNetCoreConfiguration>().